### PR TITLE
Add prefers-reduced-motion article

### PR DESCRIPTION
**Pull request by tomayac**
What's changed, or what was fixed?
- Adds prefers-reduced-motion article

**Target Live Date:** 2018-03-15

- [ ] This has been reviewed and approved by (NAME)
@jpmedley, @robdodson, @petele, @stephenmcgruer please take a look.
- [X] I have run `npm test` locally and all tests pass.
- [X] I have added the appropriate `type-something` label.
- [X] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2019/03/prefers-reduced-motion.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+## Conclusions
+
+Respecting user preferences is key for modern websites, and browsers expose more and more features
+to enable web developers to do so. The CSS Working Group are currently standardizing more
+[user preference media queries](https://drafts.csswg.org/mediaqueries-5/#mf-user-preferences)
+like `prefers-reduced-transparency` (detects if the user prefers reduced transparency),
+`prefers-contrast` (detects if the user has requested the system increase or decrease the amount of
+contrast between adjacent colors), `prefers-color-scheme` (detects if the user prefers a light or
+dark color scheme), and `prefers-reduced-transparency` (detects if the user prefers reduced
```
Comment (robdodson) [inline src/content/en/updates/2019/03/prefers-reduced-motion.md]: I think you have prefers-reduced-transparency in this list twice.

File: src/content/en/updates/2019/03/prefers-reduced-motion.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+- `no-preference`: Indicates that the user has made no preference known to the system.
+  This keyword value evaluates as `false` in the boolean context.
+- `reduce`: Indicates that the user has notified the system that they prefer an interface that
+  minimizes the amount of movement or animation, preferably to the point where all non-essential
+  movement is removed.
+
+## Working with the media query
+
+Note: `prefers-reduced-motion` is available as of Chrome Canary&nbsp;74.
```
Comment (stephenmcgruer) [inline src/content/en/updates/2019/03/prefers-reduced-motion.md]: Is it worth noting anywhere (maybe here, maybe not) that this is also available on Firefox and Safari (e.g. it's cross-browser compatible), or is that out of scope for this article?

File: src/content/en/updates/2019/03/prefers-reduced-motion.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+programmed to respect `prefers-reduced-motion`. This even works dynamically, so users can change
+their preference on-the-fly, no reload required. If a user prefers reduced motion, the non-necessary
+reveal animations are gone, and just the regular scrolling motion is left. The screencast below
+shows the demo in action:
+
+<video width="1098" height="870" intrinsicsize="1098x870" controls>
+  <source src="../../videos/2019/03/prefers-reduced-motion/prefers-reduced-motion.mp4" type="video/mp4">
+</video>
+
+## (Bonus) Forcing reduced motion on all websites
```
Comment (stephenmcgruer) [inline src/content/en/updates/2019/03/prefers-reduced-motion.md]: This section is awesome! But the suggested idea won't work for animations created via the Web Animations API, right? Might be worth noting, or maybe the 'the above hack is not guaranteed to succeed on all websites' text suffices.

File: src/content/en/updates/2019/03/prefers-reduced-motion.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+[Media Queries Level&nbsp;5](https://drafts.csswg.org/mediaqueries-5/#prefers-reduced-motion) spec.
+- `prefers-reduced-motion` on
+[Chrome Platform Status](https://www.chromestatus.com/feature/5597964353404928).
+- `prefers-reduced-motion` [Chromium bug](http://crbug.com/722548).
+- Blink
+[Intent to Implement posting](https://groups.google.com/a/chromium.org/forum/#!msg/blink-dev/NZ3c9d4ivA8/BIHFbOj6DAAJ).
+
+## Acknowledgements
+
+Massive shout-out to [Stephen McGruer](https://twitter.com/stephenmcgruer) who has implemented
```
Comment (stephenmcgruer) [inline src/content/en/updates/2019/03/prefers-reduced-motion.md]: My twitter is essentially dead (as you can see, it is filled with random contest entries and nothing else ;)). I would rather this either link to my github (https://github.com/stephenmcgruer) or just no link at all is fine.

File: src/content/en/updates/2019/03/prefers-reduced-motion.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+by using a mixture of skeleton screens, contextual metadata, and low quality image previews to
+occupy a lot of the user's time and make the whole experience *feel faster*.
+The idea is to give context to the user of what's coming and meanwhile load in things as quickly as
+possible.
+
+Finally, there are *decorative* effects like animated gradients, parallax scrolling,
+background videos, and several others.
+While many users enjoy such animations, some users dislike
+them because they feel distracted or slowed down from them. In the worst case, users may even suffer
+from motion sick as if it were a real life experience, so for these users reducing animations is a
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/content/en/updates/2019/03/prefers-reduced-motion.md]: Fixed.

File: src/content/en/updates/2019/03/prefers-reduced-motion.md (hunk 1 above)
Comment (tomayac) [reply src/content/en/updates/2019/03/prefers-reduced-motion.md]: Ooops, good catch. Also added links to all media queries.

File: src/content/en/updates/2019/03/prefers-reduced-motion.md (hunk 4 above)
Comment (tomayac) [reply src/content/en/updates/2019/03/prefers-reduced-motion.md]: I link to https://github.com/stephenmcgruer now :-D

File: src/content/en/updates/2019/03/prefers-reduced-motion.md (hunk 3 above)
Comment (tomayac) [reply src/content/en/updates/2019/03/prefers-reduced-motion.md]: Right. I have added this as a remark in parentheses.

File: src/content/en/updates/2019/03/prefers-reduced-motion.md (hunk 2 above)
Comment (tomayac) [reply src/content/en/updates/2019/03/prefers-reduced-motion.md]: Added link to https://caniuse.com/#feat=prefers-reduced-motion.

File: src/content/en/updates/2019/03/prefers-reduced-motion.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+Occasionally, the same can happen on the web: with flashing ads, fancy parallax effects, surprising
+reveal animations, autoplaying videos, etc., *the web sometimes can honestly be quite
+overwhelming*… Happily, unlike in real life, there is a solution to that. The CSS media query
+`prefers-reduced-motion` lets developers create a variant of a page for users who, well, prefer
+reduced motion. This can comprise anything from refraining from having autoplaying videos to
+disabling certain purely decorative effects, to completely redesigning a page for certain users.
+Before I dive into the feature, let's take one step back and think of what animations are used for
+on the web.
+
+## Animation on the web
```
[the remark this answers is not in the document]
Comment (tomayac) [reply src/content/en/updates/2019/03/prefers-reduced-motion.md]: I think it's important to have this background info, but I have added a "jump to the juicy bits" link now :-D

**Pull request merged**